bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tree.java

```diff
@@ -7928,8 +7928,9 @@ LRESULT wmNotifyHeader (NMHDR hdr, long wParam, long lParam) {
 							GC gc = getNewGC(nmcd.hdc, data);
 							Rectangle imageBounds = DPIUtil.autoScaleBounds(columns[i].image.getBounds(), this.getZoom(), 100);
 							int y = Math.max (0, (nmcd.bottom - imageBounds.height) / 2);
-							gc.drawImage (columns[i].image, DPIUtil.autoScaleDown(x), DPIUtil.autoScaleDown(y));
-							x += imageBounds.width + 12;
+							Rectangle scaledBounds = DPIUtil.autoScaleBounds(columns[i].image.getBounds(), getZoom(), 100);
```

Comment:
I do not understand the necessity for the `scaledBounds`. Aren't they calculated in exactly the same way in which `imageBounds` is calculated?

Reply:
Yes you are right. There has been a mistake here.